mapstory/templates/composer_new/composer.html

```diff
@@ -35,8 +35,18 @@
 
 </head>
 <body ng-controller="composerController as composer">
+  {% classification_banner %}
+  <header class="navigation" role="banner">
+    {% if CLASSIFICATION_BANNER_ENABLED %}
+      <div class="classification_banner_top_padding"></div>
+    {% endif %}
+  </header>
 
   {% include "body.html" %}
 
+  {% if CLASSIFICATION_BANNER_ENABLED and CLASSIFICATION_FOOTER_ENABLED %}
+    <div class="classification_banner_bottom_padding"></div>
+  {% endif %}
+  {% include "_classification_warning_scripts.html" %}
```

Comment:
could this `include` also be in one of the conditionals? or is it inconsequential?

Reply:
I debated that. It seemed a bit cluttered when I did it that way.
The script doesn't error or use appreciable cycles if the nodes don't exist, so I decided the clutter of more conditionals was the worse of two aesthetic evils.

Reply:
Hmm... although... I always thought it was good to put scripts like this at the bottom of the body tag, but it uses `$(document).ready()` so maybe that doesn't matter?
Pretend you're dealing with someone who hasn't done substantial HTML since 2007.